htdocs/mri/jiv/get_file.php

```diff
@@ -19,6 +19,46 @@
  *  @link     https://github.com/aces/Loris-Trunk
  */
 
+/**
+ * This function is to replace PHP's extremely buggy realpath().
+ * Taken from https://gist.github.com/umidjons/6154548
```

Comment:
What's the copyright on it? I don't see any copyright notice, which means it defaults to all rights reserved and we don't have the right to use it unless the author gives explicit written approval.

(Which makes most of my below comments a moot point..)